ibis/tests/all/test_string.py

```diff
@@ -42,19 +42,19 @@ def test_string_col_is_unicode(backend, alltypes, df):
             lambda t: t.string_col.re_search(r'[[:digit:]]+'),
             lambda t: t.string_col.str.contains(r'\d+'),
             id='re_search',
-            marks=pytest.mark.xfail_backends((Spark,)),
+            marks=pytest.mark.xfail_backends((Spark, PySpark)),
```

Comment:
How come these fail?

Reply:
PySpark doesn't support this style of regex: `[[:digit:]]+`

Reply:
Gotcha.